hg-fast-export.py

```diff
@@ -214,10 +207,10 @@ def get_branchname(name):
 
   parents = [p for p in repo.changelog.parentrevs(revision) if p >= 0]
   author = get_author(desc,user,authors)
-  if branch_name_in_commit == 'start':
-    desc = branch + '\n' + desc
-  if branch_name_in_commit == 'end':
-    desc = desc + '\n' + branch
+
+  for plugin in plugins:
+    for filter in plugin['commit_message_filters']:
+      desc = filter(desc,branch,author)
```

Comment:
When we in the future discover that we need to send more than these three values to the plugins,we'll have to update the signature for all plugins. Perhaps we should give it a `{desc: <>, branch: <>, author: <> }` dictionary instead?

Reply:
Ooh. This idea is great. Leave the signature fixed.